src/components/dashboard/dashboard.tsx

```diff
@@ -0,0 +1,363 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import BeatLoader from 'react-spinners/BeatLoader';
+import { DASHBOARD_CALLS, Get } from '../../common/dashboard';
+
+import '../analytic/analytic.scss';
+import './dashboard.scss';
+
+const Dashboard = () => {
+    const { t } = useTranslation();
+    const language = t('app.language');
+    const [userID, setUserID] = useState("cf9bfa6f-5837-42f9-9eeb-c15035c3c752");
+
+    //#region accouncements
+    const [announcementLoading, setAnnouncementLoading] = useState(true);
+    const [announcement, setAnnouncement] = useState("");
+
+    const getAnnouncement = () => {
```

Comment:
getAnnoucement has dependency in both language and userID. So it needs to be defined in useEffect event list as well.
And the function itself can be wrapped in useCallBack ( see recommendation from eslint rule: react-hooks/exhaustive-deps)